csufmse/student_information_system
# Specific pages for users and professors in /schooladmin

Testing:
* go to Users.
* select Admin, see boring page
* Select a Student, see detail tables
* select a prof see detail tables.
* guess a userid (1-20 are admin, next 100 or so are profs, rest are Students)
* hit /schooladmin/admin/<a student id>. Verify student page shows.
* Try /schooladming/professor/<a student id>. Verify student page shows
* (whatever type of id, the correct detail page should show regardless of URL)

ON REFLECTION -- maybe I should only show IF the is is the correct type...

* anyways...
* verify tables in /student and /prof are clickable.
* Verify that button labels and confirmation panels have correct user type.
